src/main/java/com/fasterxml/jackson/databind/jsontype/impl/AsDeductionTypeDeserializer.java

```diff
@@ -94,6 +103,37 @@ protected Map<BitSet, String> buildFingerprints(DeserializationConfig config, Co
         return fingerprints;
     }
 
+    private List<String> _findAliases(DeserializationConfig config, BeanPropertyDefinition property)
```

Comment:
This is just temporary(placeholder) solution for extracting aliases.
We should probably introsect at higher level, but considering for record cases also, not sure yet if there's cleaner way during StdTypeResolverBuilder construction. 🤔